systemd/fedmsg-amqp-to-zmq.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Fedmsg migration: AMQP to ZeroMQ
+After=network.target
+Documentation=https://github.com/fedora-infra/fedmsg-migration-tools
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/sysconfig/fedmsg-migration-tools
```

Comment:
Would it make more sense to have this be ``Environment="FEDMSG_MIGRATION_TOOLS_CONF=/etc/fedmsg-migration-tools/config.toml"`` and let users do ``systemctl edit fedmsg-amqp-to-zmq`` if they want to change it? comes to the same thing, but involves one fewer files

Reply:
Oh, I did not know about the edit command, that's nice. The [guidelines on that subject](https://fedoraproject.org/wiki/Packaging:Systemd#EnvironmentFiles_and_support_for_.2Fetc.2Fsysconfig_files) don't say which method is recommended. I think I like the separate file method better because we can mark it as conffile and if we want to change it or change the service file, there's no locally-edited service file to merge manually.